Sources/Tools/plFontConverter/plFontConverter.cpp

```diff
@@ -74,6 +74,13 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include "res/ui_FreeType.h"
 #include "res/ui_FreeTypeBatch.h"
 
+// Dammit Qt
+#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
+#   define QT_SKIP_EMPTY_PARTS Qt::SkipEmptyParts
+#else
+#   define QT_SKIP_EMPTY_PARTS QString::SkipEmptyParts
+#endif
+
```

Comment:
I was wondering how we were going to deal with this. Deprecated in the next to last 5.x and deleted in 6.x... Give me a break, Qt.